src/databricks/labs/ucx/assessment/aws.py

```diff
@@ -214,27 +301,104 @@ def _run_json_command(self, command: str):
             return None
         return json.loads(output)
 
+    def _run_command(self, command: str):
+        aws_cmd = shutil.which("aws")
+        code, _, error = self._command_runner(f"{aws_cmd} {command} --output json")
+        if code != 0:
+            logger.error(error)
```

Comment:
Shouldn't we throw the exception instead for it to bubble up to the top?

Reply:
I'm not sure.